ejb/WFLY-4699-expose-ejb3-deployment-info.adoc

```diff
@@ -0,0 +1,516 @@
+The plan is to add important ones as new ejb3 runtime resources, and skip those information that are
+either not commonly used, or intended for legacy features.
+
+=== Stateless: existing deployment info as runtime resources
+(http://wildscribe.github.io/WildFly/18.0//deployment/subsystem/ejb3/stateless-session-bean/index.html)
+
+* component-class-name: The component's class name.
+* declared-roles: The roles declared (via @DeclareRoles) on this EJB component.
+* execution-time: Time spend within a bean method.
+* invocations: Number of invocations processed.
+* methods: Invocation metrics per method.
+* peak-concurrent-invocations: Peak concurrent invocations.
+* pool-available-count: The number of available (i.e. not in use) instances in the pool.
+* pool-create-count: The number of bean instances that have been created.
+* pool-current-size: The current size of the pool.
+* pool-max-size: The maximum size of the pool.
+* pool-name: The name of the pool.
+* pool-remove-count: The number of bean instances that have been removed.
+* run-as-role: The run-as role (if any) for this EJB component.
+* security-domain: The security domain for this EJB component.
+* timers: EJB timers associated with the component.
+* wait-time: Time spend waiting to obtain an instance.
+
+
+=== Stateful: existing deployment info as runtime resources
+(http://wildscribe.github.io/WildFly/18.0//deployment/subsystem/ejb3/stateful-session-bean/index.html)
+
+* cache-size: Cache size.
+* component-class-name: The component's class name.
+* declared-roles: The roles declared (via @DeclareRoles) on this EJB component.
+* execution-time: Time spend within a bean method.
+* invocations: Number of invocations processed.
+* methods: Invocation metrics per method.
+* passivated-count: Passivated count.
+* peak-concurrent-invocations: Peak concurrent invocations.
+* run-as-role: The run-as role (if any) for this EJB component.
+* security-domain: The security domain for this EJB component.
+* total-size: Total size.
+* wait-time: Time spend waiting to obtain an instance.
+
+
+=== Singleton: existing deployment info as runtime resources
+(http://wildscribe.github.io/WildFly/18.0//deployment/subsystem/ejb3/singleton-bean/index.html)
+
+* component-class-name: The component's class name.
+* declared-roles: The roles declared (via @DeclareRoles) on this EJB component.
+* execution-time: Time spend within a bean method.
+* invocations: Number of invocations processed.
+* methods: Invocation metrics per method.
+* peak-concurrent-invocations: Peak concurrent invocations.
+* run-as-role: The run-as role (if any) for this EJB component.
+* security-domain: The security domain for this EJB component.
+* timers: EJB timers associated with the component.
+* wait-time: Time spend waiting to obtain an instance.
+
+
+=== Potential deployment information to be exposed for stateless, stateful & singleton:
+* jndi-names
```

Comment:
@chengfang local and remote JNDI names were in the list asked by the customer, hence it should be exposed

Reply:
the term "local" and "remote" jndi names are not proper.  This RFE exposes jndi names for different scopes: global, application-scoped, module-scoped, and they should meet user's requirement.

Reply:
@chengfang I did not understand: are or aren't we exposing local and remote JNDI names for stateless, stateful & singleton? I just want to make sure I didn't miss something important ....